kyaml/runfn/runfn_test.go

```diff
@@ -28,7 +28,10 @@ kind:
 		return
 	}
 	filter := instance.containerFilterProvider("example.com:version", "", api)
-	assert.Equal(t, &filters.ContainerFilter{Image: "example.com:version", Config: api}, filter)
+	defaultMount:= filters.StorageMount{}
+	mounts := []filters.StorageMount{}
+	mounts = append(mounts, defaultMount)
+	assert.Equal(t, &filters.ContainerFilter{Image: "example.com:version", Config: api, StorageMounts: mounts}, filter)
```

Comment:
I opted for a slightly inconsistent naming convention in this revision with 'mounts' and 'defaultMount' because 'storageMount' and 'storageMounts' ended up taking up more than 120 characters in a line.